vault.js

```diff
@@ -1,4 +1,38 @@
 'use strict';
 module.exports = function() {
+  var val = {};
   
+  return {
+      setValue: setValue,
+      getValue: getValue
+  }
+
+  function setValue(key, value){
+    console.log('value: '+value+' key: '+key+'\n');
```

Comment:
Once you're implementation is done, you should remove any extraneous console.logs